apps/reviseur/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:lts-alpine AS deps
+
+WORKDIR /app
+COPY package.json pnpm-lock.yaml pnpm-workspace.yaml .npmrc ./
+COPY apps/reviseur/package.json ./apps/reviseur/
+RUN corepack enable
+RUN pnpm install --frozen-lockfile
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)